test-monitor.cpp

```diff
@@ -2,7 +2,43 @@
 
 #include "gtest/gtest.h"
 
-TEST(Monitor, NotOkWhenAnyVitalIsOffRange) {
-  ASSERT_FALSE(vitalsOk(99, 102, 70));
-  ASSERT_TRUE(vitalsOk(98.1, 70, 98));
+std::vector<VitalValue> vitalsAllOKData = {

```

Comment:
Nice data driven approach :)
This is a declarative style of programming, which is one of the outcomes of duplicate-removal